e2e-tests/popup/review-flow/review-flow.spec.ts

```diff
@@ -0,0 +1,331 @@
+import { expect } from '@playwright/test';
+
+import { DEFAULT_SECOND_ACCOUNT, RPC_RESPONSE, URLS } from '../../constants';
+import { popup, popupExpect } from '../../fixtures';
+
+popup.describe('Popup UI:  review flow', () => {
+  popup(
+    'Positive review flow - go to web store',
+    async ({ popupPage, unlockVault, context, page }) => {
+      await unlockVault();
+
+      await popupPage.route(URLS.rpc, route =>
+        route.fulfill(RPC_RESPONSE.success)
+      );
+      await popupPage.waitForLoadState('networkidle');
+      await new Promise(r => setTimeout(r, 5000));
```

Comment:
why do you need two timeouts?

Reply:
I had some issues with failing test due to not loaded elements. Will remove unnecessary timeouts